MANUAL.md

```diff
@@ -1082,40 +1086,42 @@ const b = clone(UserSchema, a);
 
 ## Serialization
 
-As a general guide to decide between the binary format and JSON: The JSON format is great for debugging, but the binary
-format is more resilient to changes. For example, you can rename a field, and still parse binary data serialized with
+As a general guide when deciding between the binary format and JSON, the JSON format is great for debugging, but the binary
+format is more resilient to changes. For example, you can rename a field and still parse binary data serialized with
 the previous version. In general, the binary format is also more performant than JSON.
 
 ### Binary serialization options
 
+TODO: Examples to show these in context?
```

Comment:
The options change how binary data is serialized. To give an example, we have to show binary data. I'm afraid we would need to do an excursion to explain the binary format first to effectively show that. Or we would need a complex setup with multiple example files and many steps. It would probably be about one page of code per option.